sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/LargeITCluster/data-load.sh

```diff
@@ -87,6 +116,5 @@ echo "-----------------------------"
 ${recordcount} -p insertorder=ordered -p fieldlength=20 -P workloads/workloadd \
 -s > workloada_load_res.txt
 
-# Delete cassandra sinle node set up
-cd ../LargeITCluster
-kubectl delete -f cassandra-svc-temp.yaml
+# Delete cassandra single node set up before exit 
+trap delete_service EXIT
```

Comment:
I think you might have to put this up at the beginning of the file (maybe just after the function is defined?), otherwise I don't know that it works